includes/admin/class-wp-job-manager-admin.php

```diff
@@ -58,7 +65,14 @@ public function __construct() {
 		include_once dirname( __FILE__ ) . '/class-wp-job-manager-cpt.php';
 		WP_Job_Manager_CPT::instance();
 
-		include_once dirname( __FILE__ ) . '/class-wp-job-manager-promoted-jobs-admin.php';
+		/**
+		 * Documented in class-wp-job-manager.php
+		 */
+		$this->are_promoted_jobs_enabled = apply_filters( 'job_manager_enable_promoted_jobs', true );
```

Comment:
I'm thinking it's fine to just call the filter in multiple places, no need for a new class property for this.

Reply:
Hmm I am not sure I agree with this one. It will be easier to update when we change the default value. Also we will probably need to repeat the comments too if we want all filters to be documented.